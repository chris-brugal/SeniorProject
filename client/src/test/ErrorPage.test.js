import React from 'react';
import ErrorPage from '../components/ErrorPage';
import renderer from "react-test-renderer";

describe("snapshot testing", () => {

test("snapshot testing for Error Page component", () => {
    const comp = renderer.create(<ErrorPage />).toJSON()
    expect(comp).toMatchSnapshot()
})
})