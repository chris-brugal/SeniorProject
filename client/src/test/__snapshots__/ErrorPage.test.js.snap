// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing snapshot testing for Error Page component 1`] = `
<div>
  ErrorPage
</div>
`;
